vital/logger/tests/test_logger.cxx

```diff
@@ -99,7 +99,14 @@ IMPLEMENT_TEST(check_levels)
 // ------------------------------------------------------------------
 IMPLEMENT_TEST(logger_factory)
 {
+  // Need to unset any logger factory specification so we will use the
+  // default factory
+#if defined _WIN32
+  _putetenv( "VITAL_LOGGER_FACTORY=" );
```

Comment:
@linus-sherrill there is a typo. `_putetenv` should be `_putenv`.  After that change it works.  Can you make the correction since the PR comes from your account?